tools/vf-component-library/src/site/_includes/layouts/base.njk

```diff
@@ -6,6 +6,31 @@
     {# <link rel="stylesheet" media="all" href="https://dev.assets.emblstatic.net/vf/develop/css/styles.css" /> #}
     <link rel="stylesheet" media="all" href="{{ '/css/styles.css' | url }}" />
     <title>{{ pagination.items[0].name or title or (renderData and renderData.title)}} | {{ siteConfig.siteInformation.title }}</title>
+    {% render '@vf-favicon' %}
+    <!-- Descriptive meta -->
+    <meta name="title" content="{{ title or (renderData and renderData.title) or siteConfig.siteInformation.title}}">
+    <meta name="author" content="{{ meta.author or "EMBL" }}">
+    <meta name="robots" content="index, follow">
+    <meta name="keywords" content="{{ meta.keywords }}">
+    <meta name="description" content="{{ meta.description or subtitle}}">
+
+    <!-- Open Graph / Facebook -->
+    <meta property="og:type" content="website">
+    <meta property="og:url" content="{{siteConfig.siteInformation.url}}{{ page.url }}">
+    <meta property="og:title" content="{{ title or (renderData and renderData.title) or siteConfig.siteInformation.title}}">
```

Comment:
Isn't `siteConfig.siteInformation.title` and such specific to `vf-eleventy`?

Reply:
Yeah, this is just for the Eleventy based component library.

Reply:
🤦‍♂️

Reply:
😄